app/models/service/dialog_properties/retirement.rb

```diff
@@ -51,7 +48,10 @@ def retire_on_date=(value)
 
       def retirement_warning
         warn_value = parse_retirement_warn
-        @attributes[:retirement_warn] = warn_value if warn_value
+        if warn_value
+          days_between_warn_and_retirement = (@attributes[:retires_on] - warn_value)/(60*60*24)
```

Comment:
Should we make `86400` (60\*60\*24) some kind of constant (or does Rails already have one)? Or should we use something like `1.day.seconds.to_i`?

Either way I think the 60\*60\*24 should be a bit more descriptive even though I would imagine most people could figure out what it means based on the numbers.